### ISSUE TITLE
Ebean migrations can not be executed if they contain ';' into a string

### ISSUE THREAD

**Issue by romannikov** [ebean-orm/ebean#1004]
## Expected behavior
Migration should not fail on a string with ; sign into it

## Actual behavior
Migration fails on a string with ; sign into it

### Steps to reproduce
Create migration with following code:
```console
CREATE TABLE test {
  id integer NOT NULL,
  val text;
}
INSERT INTO test (id, val) VALUES (1, 'some lovely text with ; unexpected sign');
```
and try to execute a migration

**Comment by slmsbrhgn** [comment]
You example does not work in Mysql or Postgresql (using `{ }` in create table, `;` after column definition). Fix that and see if you still have the same problem.

**Comment by romannikov** [comment]
yes, you're right. Thanks for pointing to the problem with sql syntax. 
The correct sql for a table creation and testing is following:
```
CREATE TABLE test (
  id integer NOT NULL,
  val text
);
INSERT INTO test (id, val) VALUES (1, 'some lovely text with ; unexpected sign');
```
but it's not the main problem. The main problem is in the insert statement with ';' char into a string.

**Comment by slmsbrhgn** [comment]
What happens if you replace the `;` inside the string with `\\;` or `\;` ?

**Comment by rbygrave** [comment]
This is being fixed in ebean-dbmigration as https://github.com/ebean-orm/ebean-dbmigration/issues/26

**Comment by rbygrave** [comment]
This just updates the ebean-dbmigration dependency to the newly released 10.1.8
